Source/WebCore/dom/EventLoop.h

```diff
@@ -208,6 +210,12 @@ class EventLoopTaskGroup : public CanMakeWeakPtr<EventLoopTaskGroup> {
     EventLoopTimerHandle scheduleRepeatingTask(Seconds nextTimeout, Seconds interval, TaskSource, EventLoop::TaskFunction&&);
     void removeRepeatingTimer(EventLoopTimer&);
 
+    void setTimerAlignment(EventLoopTimerHandle, TimerAlignment&);
+    void didChangeTimerAlignmentInterval(EventLoopTimerHandle);
+    void setTimerHasReachedMaxNestingLevel(EventLoopTimerHandle, bool);
+    void augmentTimerNextTimeout(EventLoopTimerHandle, Seconds delta);
```

Comment:
Not sure that "augment" is the best word: I have to stop and think about whether it means "replace" or "extend". Maybe "extend" is the right term to use?

Also, "timeout" => "fire time".

Reply:
Yeah, adjustTimerFireTime?